.github/workflows/cmake.yml

```diff
@@ -63,6 +65,10 @@ jobs:
         wget -O ${{github.workspace}}/dpcpp_compiler.tar.gz https://github.com/intel/llvm/releases/download/sycl-nightly%2F20230626/dpcpp-compiler.tar.gz
         tar -xvf ${{github.workspace}}/dpcpp_compiler.tar.gz
 
+    - name: Setup DPC++
```

Comment:
why do we need this, actually? if it's only needed for fuzzing, please add an `if` here (and perhaps add a comment on the purpose of this)

Reply:
It's needed for building kernel binaries with sycl. Removed from the path not running fuzztests.